analyses/snv-frequencies/01-snv-frequencies.R

```diff
@@ -629,29 +665,63 @@ htl_df <- read_tsv('../../data/histologies.tsv', guess_max = 100000,
                    col_types = cols(.default = col_guess()))
 
 htl_df <- htl_df %>%
-  filter(!experimental_strategy == "Methylation")
+  filter(!experimental_strategy == "Methylation") %>%
+  # want panel data to be a distinct cohort from whole genome or whole exome data,
+  # so changing the cohort column to include "panel" for panel data
+  # rename(cohort_hist = cohort)
+  mutate(cohort = case_when(cohort == 'TARGET' & 
+                              experimental_strategy == 'Targeted Sequencing' ~
+                              'TARGET panel',
+                            cohort == 'DGD' & 
+                              experimental_strategy == 'Targeted Sequencing' ~ 
+                              'CHOP P30 panel',
+                            TRUE ~ cohort))
+
+# The cohort names we want to display on the MTP website do not match the 
+# PedcBio cohort names, so need a 2-column table matching them up with 2 columns:
+# - cohort: the cohort we want displayed on MTP
+# - pedcbio_cohort: the name of that cohort on PedcBio
+htl_df %>%
+  distinct(cohort) %>%
+  mutate(pedcbio_cohort = case_when(cohort == 'TARGET panel' ~ 'TARGET',
+                                    cohort == 'CHOP P30 panel' ~ 'DGD',
```

Comment:
```suggestion
  mutate(cohort = case_when(cohort == 'TARGET' & 
                              experimental_strategy == 'Targeted Sequencing' ~
                              'TARGET Panel',
                            cohort == 'DGD' & 
                              experimental_strategy == 'Targeted Sequencing' ~ 
                              'CHOP P30 Panel',
                            TRUE ~ cohort))

# The cohort names we want to display on the MTP website do not match the 
# PedcBio cohort names, so need a 2-column table matching them up with 2 columns:
# - cohort: the cohort we want displayed on MTP
# - pedcbio_cohort: the name of that cohort on PedcBio
htl_df %>%
  distinct(cohort) %>%
  mutate(pedcbio_cohort = case_when(cohort == 'TARGET Panel' ~ 'TARGET',
                                    cohort == 'CHOP P30 Panel' ~ 'DGD',
```

Hi @kelseykeith can you just update this to be consistent with fusion-freq/ capitalization style. Thanks!